docs/conf/prague/2018/hike.rst

```diff
@@ -0,0 +1,77 @@
+:template: {{year}}/{{templatecode}}/generic.html
+:banner: _static/2018/assets/headers/hike.png
+
+Write the Docs Hike
+===================
+
+Every year we have a conference hike, and at this point it's a fantastic tradition.
+We'll be doing the same hike again this year, because it's the best one easily accessible from downtown Portland.
+
+It's rained on us the past, but we have faith it will be beautiful this year! We shall see Mount Hood at the top :)
+
+Logistics
+---------
+
+- Date & Time: Leaves promptly at **Saturday, May 5, 2 PM**. Meet at **1:45**.
```

Comment:
should we variable-ify the date here?

Reply:
Less important, cos the boat only happens in Prague.

Reply:
I'm actually gonna move both to `outing.rst` and conditional include. Butnext time.

Reply:
Done!